vm/hash.go

```diff
@@ -231,6 +635,59 @@ func (h *HashObject) length() int {
 	return len(h.Pairs)
 }
 
+func (h *HashObject) sortedKeys() []string {
+	var arr []string
+	for k := range h.Pairs {
+		arr = append(arr, k)
+	}
+	sort.Strings(arr)
+	return arr
+}
+
+func (h *HashObject) isEqualTo(compare *HashObject) bool {
```

Comment:
Can we use [reflect.DeepEqual](https://golang.org/pkg/reflect/#DeepEqual) in this case?